licenses/ARCHIVER-UTILS_MIT.TXT

```diff
@@ -1,22 +1,22 @@
-Copyright (c) 2015 Chris Talkington.
-
-Permission is hereby granted, free of charge, to any person
-obtaining a copy of this software and associated documentation
-files (the "Software"), to deal in the Software without
-restriction, including without limitation the rights to use,
-copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the
-Software is furnished to do so, subject to the following
-conditions:
-
-The above copyright notice and this permission notice shall be
-included in all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
-OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
-HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
-WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+Copyright (c) 2015 Chris Talkington.

```

Comment:
@helio-frota Hi! Any idea why lots of the files here have changed here with just whitespace?

Reply:
@grdryn good question, I have no idea :man_shrugging: 
We are just making a copy by reading the stream and piping to another file (and location), without code interference:
https://github.com/bucharest-gold/license-reporter/blob/master/lib/modules/report.mdl.js#L58-L59

Reply:
just to be sure, the files remains unchanged
https://github.com/bucharest-gold/license-reporter/issues/270#issuecomment-359009993

Reply:
@helio-frota I wonder if it could be an issue with different platforms/environments or Nodejs/npm combinations or something like that?

Reply:
good question, I have an windows 10 installed here, I'm going to test, also with diff node versions.